frontend/src/components/TaskTable.tsx

```diff
@@ -255,28 +93,6 @@ export const TaskTable: React.FC<{
     );
   };
 
-  const tableHeaders: TableHeader[] = [
-    {
-      label: 'Status',
-      sorting: SortingTypes.SORT_STATUS,
-      textAlign: 'center',
-      width: '1em',
-    },
-    { label: 'Title', sorting: SortingTypes.SORT_NAME },
-    { label: 'Description', sorting: SortingTypes.SORT_DESC },
-    { label: 'Waiting for ratings', sorting: SortingTypes.NO_SORT },
-    {
-      label: 'Rating',
-      sorting: SortingTypes.SORT_RATINGS,
-      width: '1em',
-    },
-    {
-      label: 'Created on',
-      sorting: SortingTypes.SORT_CREATEDAT,
-      width: '8em',
-    },
-  ];
-
   const renderUnratedTasks = () => {
```

Comment:
This function is used for both unrated and rated tasks, so maybe renderTasks would be a more suiting name